charts/comet-site-v8/templates/build-cron-job.yaml

```diff
@@ -38,7 +38,13 @@ spec:
           serviceAccountName: {{ include "comet-site.fullname" . }}
           containers:
             - name: {{ .Chart.Name }}-build
+              securityContext:
+                {{- toYaml .Values.builder.securityContext | nindent 16 }}
```

Comment:
Not part of this PR

Reply:
moved to this PR: https://github.com/vivid-planet/comet-charts/pull/39